### DISCUSSION TITLE
This is the original dashboard design model

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/RadioLogger#6] [General]
This is the dashboard design

![Screenshot_2022-09-11_at_22-18-03_Radio_Logger-2](https://user-images.githubusercontent.com/47549872/232289521-7e8522b5-cab6-4475-854d-e77846a27fea.png)

**Comment by gbowne1** [comment]
![Screenshot_2022-09-29_at_22-08-27_Radio_Logger](https://github.com/gbowne1/RadioLogger/assets/47549872/e9b6c7c4-dfb7-4de2-b10d-98be44972845)

**Reply by gbowne1** [reply]
I have screen shots of when I was trying to use React on this project.  The design looked a lot better.

@jzunigarce I added your ESLint and Prettier to both client and server.

**Reply by jzunigarce** [reply]
On server i add configuration for eslint and prettier, I haven't uploaded it yet

**Reply by gbowne1** [reply]
There is already config in the root.

**Reply by gbowne1** [reply]
I tried running prettier on the code.  Theres abour 90 files it will clean up.

**Comment by gbowne1** [comment]
Profile Design Model

![Screenshot_2022-09-28_at_15-13-48_Radio_Logger](https://github.com/gbowne1/RadioLogger/assets/47549872/8fa92093-543a-4d14-bfc7-05a805c42336)

**Reply by jzunigarce** [reply]
Good,we need add more field to user model.

**Reply by gbowne1** [reply]
This Profile design needs only a few things added to it.

Should be accessible from the user Icon in the top nav

**Reply by gbowne1** [reply]
I never finished this html in the old project.  But this is the basic design I was going for.  I also want to make it shsreable like on facebook and twitter and qrz.com

**Reply by gbowne1** [reply]
I found the original template for this.

**Reply by jzunigarce** [reply]
Can you upload?

**Reply by gbowne1** [reply]
yeah.  The UI needs a lot of help..  I got some done but you can see from the pics the idea I was going for. Design (aesthetics) is not my strong point, I just know what goes where and its functionality.

**Reply by gbowne1** [reply]
@jzunigarce I just pushed the profile update to /profile/profile.html
